From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Charles Kerr <charles@charleskerr.com>
Date: Wed, 19 Jul 2023 15:57:19 -0500
Subject: fix ftbfs Werror Wunreachable-code-break

Fix a deps warning that causes FTBFS when building with -Werror.

This patch can be removed if fixed upstream.
Upstream PR 1: https://github.com/ada-url/ada/pull/464
Upstream PR 2: https://github.com/ada-url/idna/pull/31

diff --git a/deps/ada/ada.cpp b/deps/ada/ada.cpp
index 570a659d118de5404003d0cd3731523d863de16b..43147d495f78eea916f895258f9c1846bd55f40a 100644
--- a/deps/ada/ada.cpp
+++ b/deps/ada/ada.cpp
@@ -2829,7 +2829,6 @@ std::u32string map(std::u32string_view input) {
         break;
       case 2:
         return error;  // disallowed
-        break;
 
       // case 3 :
       default:
@@ -15175,4 +15174,4 @@ const ada_url_components* ada_get_components(ada_url result) noexcept {
 }
 }  // extern "C"
 /* end file src/ada_c.cpp */
-/* end file src/ada.cpp */
+/* end file src/ada.cpp */
\ No newline at end of file
